install-jellyfin.ps1

```diff
@@ -51,33 +51,48 @@ else
 
 if($Quiet.IsPresent -or $Quiet -eq $true){
     $Script:JellyfinDataDir = "$env:AppData\jellyfin\"
-    if([string]::IsNullOrEmpty($InstallLocation)){$Script:DefaultJellyFinInstallDirectory = "$env:Appdata\jellyfin\"}else{$Script:DefaultJellyFinInstallDirectory = $InstallLocation}
+    if([string]::IsNullOrEmpty($InstallLocation)){
+        $Script:DefaultJellyfinInstallDirectory = "$env:Appdata\jellyfin\"
+    }else{
+        $Script:DefaultJellyfinInstallDirectory = $InstallLocation
+    }
     
-    if([string]::IsNullOrEmpty($EmbyLibraryLocation)){$Script:defaultEmbyDataDir = "$env:Appdata\Emby-Server\data\"}else{$Script:defaultEmbyDataDir = $EmbyLibraryLocation}
+    if([string]::IsNullOrEmpty($EmbyLibraryLocation)){
+        $Script:defaultEmbyDataDir = "$env:Appdata\Emby-Server\data\"
+    }else{
+        $Script:defaultEmbyDataDir = $EmbyLibraryLocation
+    }
     
-    if($InstallAsService.IsPresent -or $InstallAsService -eq $true){$Script:InstallAsService = $true ; $Script:JellyfinDataDir = "C:\WINDOWS\system32\config\systemprofile\AppData\Roaming\jellyfin\" }else{$Script:InstallAsService = $false}
-    if($null -eq $ServiceUser){$Script:InstallServiceAsUser = $false}else{$Script:InstallServiceAsUser = $true; $Script:UserCredentials = $ServiceUser;  $Script:JellyfinDataDir = "C:\Users\$($Script:UserCredentials.UserName)\Appdata\Roaming\jellyfin\"}
+    if($InstallAsService.IsPresent -or $InstallAsService -eq $true){
+        $Script:InstallAsService = $true 
+        $Script:JellyfinDataDir = "C:\WINDOWS\system32\config\systemprofile\AppData\Roaming\jellyfin\" 
+    }else{$Script:InstallAsService = $false}
+    if($null -eq $ServiceUser){
+        $Script:InstallServiceAsUser = $false
+    }else{
+        $Script:InstallServiceAsUser = $true
+        $Script:UserCredentials = $ServiceUser
+        $Script:JellyfinDataDir = "C:\Users\$($Script:UserCredentials.UserName)\Appdata\Roaming\jellyfin\"}
```

Comment:
this looks a bit dangerous... what would happen if my users aren't in `C:\Users`? I know that probability is very low but I think it's possible...

Reply:
A suggestion could be using something like `%USERPROFILE%\Appdata\Roaming\jellyfin`

Reply:
This needs to be fixed - don't assume `C:\` exists.